src/simulation_lifecycle/simulation.c

```diff
@@ -15,6 +32,7 @@ int check_sim_preconditions(const char *model, const feature_set_t *p_features,
     } else if (!model_exists(model)) {
         return MODEL_NOT_FOUND;
     }
+    return 0;
```

Comment:
Why returning 0 here?

Reply:
Oops sorry. I did that in order to try different stuff and forgot to remove it.